input.js

```
@@ -0,0 +1,25 @@
+const http = require('http');
+const fs = require('fs');
+
+const PORT = 8080;
+
+// ⚠️ Hardcodeando credenciales (mala práctica)
+const username = 'admin';
+const password = '123456';
+
+// ⚠️ Usando eval (muy inseguro)
+function runUserCode(input) {
+    return eval(input);  // Esto puede ejecutar cualquier cosa
```

Biome (1.9.4)

[error] eval() exposes to security risks and performance issues.

See the MDN web docs for more details.
Refactor the code so that it doesn't need to call eval().

(lint/security/noGlobalEval)